docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+---
+version: '3.3'
+
+services:
+  dev_ringer:
+    build:
+      context: .
+    image: dev-ringer
+    ports:
+      - "8081:8081"
+      - "8082:8082"
+      - "8444:8444"
```

Comment:
This works alright for this specific example, but every drp configuration is different, so we'd have to specify all of the ports in the drp? Is there a more dynamic way we could do this?